package.json

```diff
@@ -42,24 +42,24 @@
     "@types/nunjucks": "^3.2.6",
     "@types/require-directory": "^2.1.6",
     "@types/serve-favicon": "^2.5.7",
-    "applicationinsights": "^2.9.7",
+    "applicationinsights": "^2.9.8",
     "autobind-decorator": "^2.4.0",
     "awilix": "^7.0.3",
-    "axios": "1.12.2",
-    "cacache": "^19.0.1",
+    "axios": "1.13.2",
+    "cacache": "^20.0.3",
     "config": "^3.3.12",
     "cookie-parser": "^1.4.7",
     "csrf-csrf": "^3.2.2",
-    "express": "4.21.2",
+    "express": "4.22.1",
     "express-nunjucks": "^3.1.2",
-    "glob-parent": "6.0.2",
+    "glob-parent": "^6.0.2",
     "govuk-frontend": "^5.10.2",
     "helmet": "^4.6.0",
     "http-proxy-middleware": "^2.0.9",
     "i18next": "^21.10.0",
     "i18next-http-middleware": "^3.7.4",
     "jquery": "^3.7.1",
-    "js-yaml": "^4.1.0",
+    "js-yaml": "^3.14.2",
```

Comment:
Note 3.14.2 is required by a dev resolution for a cve, hence having both this and 4.1.0 makes the package manager highlight it as redundant